querybook/webapp/redux/dataDoc/reducer.ts

```diff
@@ -20,8 +20,32 @@ const initialState: Readonly<IDataDocState> = {
     sessionByDocId: {},
     editorsByDocIdUserId: {},
     accessRequestsByDocIdUserId: {},
+    dataDocWithSchema: {
```

Comment:
the name dataDocWithSchema is not very clear to me, what is the "schema" here? You could just name it scheduledDocs or docDocWithSchedule